languageserver/integration/commands.go

```diff
@@ -86,42 +97,36 @@ func (i *FlowIntegration) commands() []server.Command {
 //   * the arguments, encoded as JSON-CDC
 func (i *FlowIntegration) submitTransaction(conn protocol.Conn, args ...interface{}) (interface{}, error) {
 
-	err := server.CheckCommandArgumentCount(args, 2)
+	err := server.CheckCommandArgumentCount(args, 3)
 	if err != nil {
 		return nil, err
 	}
 
-	uri, ok := args[0].(string)
-	if !ok {
-		return nil, fmt.Errorf("invalid URI argument: %#+v", args[0])
-	}
-
-	doc, ok := i.server.GetDocument(protocol.DocumentUri(uri))
+	_, ok := args[1].([]interface{})
 	if !ok {
-		return nil, fmt.Errorf("could not find document for URI %s", uri)
+		return nil, fmt.Errorf("invalid transaction arguments: %#+v", args[1])
 	}
 
 	rawTransactionArguments, ok := args[1].([]interface{})
 	if !ok {
-		return nil, fmt.Errorf("invalid transaction arguments: %#+v", args[1])
+		return nil, fmt.Errorf("invalid script arguments: %#+v", args[1])
```

Comment:
Why the change, this function is submitting a transaction, so the argument contains the transaction arguments, no?

Reply:
My bad, copy pasted portion of code from script execution and missed that later 🙇‍♂️